webapp/src/test/java/com/box/l10n/mojito/service/assetExtraction/MultiBranchStateBlobStorageTest.java

```diff
@@ -58,6 +59,8 @@ public void test() {
         multiBranchStateBlobStorage.getMultiBranchStateForAssetExtractionId(
             assetExtractionId, version);
     assertThat(multiBranchStateForAssetExtractionId.get())
-        .isEqualToComparingFieldByField(multiBranchState);
+        .usingRecursiveComparison()
+        .withComparatorForType(Comparator.comparing(ZonedDateTime::toInstant), ZonedDateTime.class)
```

Comment:
why is this needed?

Reply:
isEqualToComparingFieldByField is deprecated and usingRecursiveComparison does a similar function.
I had to change the comparator for ZonedDateTime to compare the instant instead, as the ZonedDateTime objects were not always the same even though the instant were.

Reply:
there is no native support from AssertJ? do we need a newer version?